.github/workflows/10-alpine3.11.yml

```diff
@@ -0,0 +1,17 @@
+name: 10 on alpine3.11
+
+on:
+  pull_request:
+    paths:
+      - 10/alpine3.11/Dockerfile
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: sudo apt-get install bats
```

Comment:
Maybe need to add `-y` here for apt?

Reply:
Doesn't seem to be needed when I was testing